atoms/LoadingProgress/index.js

```diff
@@ -15,6 +15,10 @@ export default React.createClass({
     return style
   },
 
+  shouldComponentUpdate: function shouldComponentUpdate (newProps) {
```

Comment:
Yes I do :) My copy in `node_modules` (which I was using for testing) has old code which used that syntax. I'll update it, thank you :)